samples/helloworld/README.md

```diff
@@ -508,6 +510,18 @@ ninja
 ninja run
 ```
 
+### Mixing LVI Mitigation
+
+You can use different LVI mitigation method to build your app than OpenEnclave, however some additional cmake configuration is required. See below table for all possible combination.
```

Comment:
```suggestion
You can use a different LVI mitigation method to build your app than Open Enclave, however additional cmake configuration may be required. See below table for all possible combination.
```

From your table there are no additional requirements when building OE with Clang and application with GNU so it is not required for all cases.